capella/versions/6.1.0/dropins.yml

```diff
@@ -8,24 +8,35 @@ dropins:
     eclipseRepository: 'https://download.eclipse.org/capella/addons/xhtmldocgen/updates/releases/6.1.0/'
     installIU:
       - 'org.polarsys.capella.docgen.package.feature.feature.group'
+  DarcViewpoint:
```

Comment:
Also add the dropin to the list of supported add-ons: https://github.com/DSD-DBS/capella-dockerimages/blob/main/docs/docs/capella/base.md?plain=1#L100